spring-data-mongodb/src/main/java/org/springframework/data/mongodb/repository/Query.java

```diff
@@ -63,6 +63,14 @@
 	boolean count() default false;
 
 	/**
+	 * Returns whether the query defined should be executed as exists projection.
+	 *
+	 * @since 1.10
+	 * @return
+	 */
+	boolean exists() default false;
```

Comment:
I wonder if we could derive that from the return type of the query method instead. Queries on MongoDB *always* return a document which can't be mapped to a `boolean` naturally. So we could just enable the exists execution for all query methods that return a `boolean`, can't we?